src/createServer.js

```diff
@@ -1,3 +1,61 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  return http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const url = new URL(request.url, `http://${request.headers.host}`);
+    const text = url.pathname.slice(1);
+    const toCase = url.searchParams.get('toCase');
+
+    const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message: 'Text to convert is required. Correct request is: '
+          + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: '"toCase" query param is required. Correct request is: '
+          + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (!validCases.includes(toCase)) {
+      errors.push({
+        message: 'This case is not supported. Available cases: '
+          + 'SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
```

Comment:
for better maintainability instead of using the hardcoded string "SNAKE, KEBAB, CAMEL, PASCAL, UPPER." you should use validCases array. This ensures that the list of valid cases is consistent and easier to update